static/js/notifications.js

```diff
@@ -513,6 +513,37 @@ function get_message_header(message) {
     return "PM with " + message.display_reply_to;
 }
 
+exports.check_local_echo_in_view = function () {
+    var focused_table_rec_rows = $('.focused_table .recipient_row');
+    // If the message is first in its narrow
+    if (focused_table_rec_rows.length === 0) {
+        return true;
+    }
+    var echo = $(focused_table_rec_rows[focused_table_rec_rows.length - 1]).children('.message_row');
```

Comment:
You can use `rows.get()` and similar functions to get this (since you have the local message ID of the message)